### ISSUE TITLE
Window not behaving and setting icon not working with multiple screens on

### ISSUE THREAD

**Issue by Hibge** [LISTEN-moe/desktop-app#17]
When having more than three 1920x1080 screens the window shrinks and I am unable to open the settings icon to login/change between Kpop-Jpop.
![Capture](https://user-images.githubusercontent.com/59215013/71429878-d202b780-2703-11ea-88a7-ad9b1a0e00d8.JPG)
I have reverted back to a older version (0.2.2) which still allows me to to change the song client but times out sometimes.
Also, with 4 monitors the window gets abit smaller (approx. 10% smaller than attached picture)
Even by disabling all screens but one, same issue happens as to that I am unable to access the setting icon.

Current system:
Windows 7 & 10 Pro - latest version 
LISTEN Version: 0.2.6
Windows 7 x3 screens (1920x1080)
Windows 10 x4 screens (1920x1080)

**Comment by RandomPotato26** [comment]
click edge and drag to expand, the setting icon had a few issues with me as well but it was fixed with a fresh install of it 🤷‍♂  
![x3LwZFWxJT](https://user-images.githubusercontent.com/15883679/71505407-ee047700-2831-11ea-9a7b-64c49fcb13fd.gif)

**Comment by Hibge** [comment]
Thankyou, that worked (click the edge and expand) but the setting icon is still not functioning.

**Comment by robflop** [comment]
@Hibge can you still reproduce this on newest version (currently 0.3.0) or is it fixed?

**Comment by Hibge** [comment]
@robflop The issue has gotten worse, where I cannot see the window tab of the client.
I can still access the control panel of the client from the taskbar. (Have the latest 0.3.0)

The picture shows 2 monitors as my third broke, but the window is not shown anywhere.
![listen](https://user-images.githubusercontent.com/59215013/89622283-64d41900-d8c5-11ea-8d09-d7be22c74db5.jpg)

I have connected too my monitors and used the new client on other PC's with no issues.

Have cleared any/all temp/cache files with no results. 
Also have disconnected one, changed resolution, added a 3 third monitor - still no window shown.
The issue is only on one of my windows 7 PC's and neither the older clients are showing now. Unsure why

If its to much of an unknown, I can still continue using the web/app version.

**Comment by robflop** [comment]
Okay, thanks for the update. I sadly can't help further as I don't own a 2nd monitor myself.

**Comment by Hibge** [comment]
Hello @robflop,
I have managed to fix my previous issue of where the window of the application doesn't show up.

I had to go to C:/Users/(user name)/AppData/Roaming, and had to delete the older files from previous versions of the application.
Also there were 2 folders in the C:/Users/(user name)/AppData/Local, which I had to delete.

After deleting these older files of the listen.moe application. 
I installed the newer version 0.3.0 and everything is now working, including the settings button and all.
